docs/SSH.md

```diff
@@ -1,47 +1,83 @@
-SSH stands for secure shell. You can remotely connect to the raspberry pi terminal with an SSH client. A popular ssh client is [Putty](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html) for Windows. The `ssh` command can be used on macOS and GNU/Linux.
+Enabling SSH (Secure Shell) allows remote connection to the Raspberry Pi. This provides a means for adding roms, changing configuration, updates, and other convienient procedures by using SSH and SFTP clients to connect to the Raspberry Pi from anywhere in the world. For this instruction and for the sake of simplicity, this guide assumes that both the Raspberry Pi and the computer you are using to remotely connect to it are on the same local network.
```

Comment:
I'd leave out the 'anywhere in the world' comment, it's too generous for what SSH access can do by default. Local access is fine.